### PR TITLE
feat: use legacy server to fetch avatar data

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: packages/shared/index.ts [hunk 1]
```diff
@@ -111,7 +120,17 @@ export async function initShared(container: HTMLElement): Promise<ETHEREUM_NETWO
     }
 
     let spec: ProfileSpec
-    if (response && response.ok) {
+    if (response && !response.ok) {
```
Comment (moliva) [inline packages/shared/index.ts]:
```suggestion
    if (!response || !response.ok) {
```

File: packages/shared/world/profiles.ts [hunk 2]
```diff
@@ -15,12 +15,16 @@ export async function resolveProfile(uuid: string = ''): Promise<Profile> {
...
 
   let spec: ProfileSpec
-  // FIXME - forcing random generation until profile migration is finished - moliva - 29/07/2019
-  if (false && response && response.ok) {
+  if (response && response.ok) {
     // @ts-ignore
     spec = (await response.json()) as ProfileSpec
   } else {
-    spec = await createStubProfileSpec()
+    const legacy = await fetchLegacy(uuid)
```
Comment (moliva) [inline packages/shared/world/profiles.ts:22]: We shouldn't do this on `PREVIEW` mode as the user is not logged in.

File: packages/shared/index.ts [hunk 3]
```diff
@@ -111,7 +120,17 @@ export async function initShared(container: HTMLElement): Promise<ETHEREUM_NETWO
...
+      const legacy = await fetch(`${getServerConfigurations().avatar.server}api/profile`, {
```
Comment (moliva) [inline packages/shared/index.ts]: Could we use `fetchLegacy` here?

File: packages/shared/index.ts (hunk 1 above)
Comment (eordano) [reply packages/shared/index.ts]: See line 133?

Comment (moliva) [reply packages/shared/index.ts]: I mean, what happens if prior to this `fetchProfile` erred and no `response` is set due to an issue. We could still try and fetch from the legacy system.

Comment (eordano) [reply packages/shared/index.ts]: Thanks!

**Pull request merged**